format/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>org.apache.thrift</groupId>
       <artifactId>libthrift</artifactId>
-      <version>0.9.3</version>
+      <version>0.19.0</version>
```

Comment:
It is better we do this in another PR. I am preparing a PR for this.

Reply:
ok